scm-info/javax/activation/_artifact/activation/1.1.1/scm.yaml

```diff
@@ -0,0 +1,3 @@
+---
+type: "git"
+uri: "https://github.com/rnc/glassfish-jaf"
```

Comment:
Can you create this under: https://github.com/orgs/jvm-build-service-code

Reply:
Sure, didn't know about that repo. Transferred ownership of the repo to there.